jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLambdaBuilderTest.kt

```diff
@@ -130,4 +131,24 @@ class JavaLambdaBuilderTest : BaseLambdaBuilderTest() {
             "lib/aws-lambda-java-core-1.2.0.jar"
         )
     }
+
+    @Test
+    fun unsupportedSystem() {
+        val handlerPsi = projectRule.fixture.addClass(
+            """
+            package com.example;
+
+            public class SomeClass {
+                public static String upperCase(String input) {
+                    return input.toUpperCase();
+                }
+            }
+            """.trimIndent()
+        )
+
+        assertThatThrownBy {
+            buildLambda(projectRule.module, handlerPsi, Runtime.JAVA8, "com.example.SomeClass")
+        }.isInstanceOf(IllegalStateException::class.java)
+            .hasMessageEndingWith("is not managed by Maven or Gradle")
```

Comment:
Hmm, as the exception message is managed by the localization properties, I'm wondering if it is appropriate to test the exception message

Reply:
Ya it is fragile, but it does enforce the correct exception is thrown and not a random different IllegalState

Reply:
I understand you want to enforce the correct exception. It's better to fix the fragile test, but I'm fine with it too considering we are not expanding the localization in a short term.

Reply:
I can just use the message() call here as well. I'll make the change tomorrow